docs/generate_format_table.py

```diff
@@ -0,0 +1,159 @@
+import large_image
+from pathlib import Path
+from format_examples_datastore import EXAMPLES_FOLDER, format_examples, fetch_all
+
+
+TABLE_FILE = Path('./format_table.rst')
+NO_MULTIFRAME_SOURCES = ['deepzoom', 'openjpeg', 'openslide']
+
+
+def evaluate_examples():
+    large_image.tilesource.loadTileSources()
+    available_tilesources = large_image.tilesource.AvailableTileSources
+    results = []
+    for format_data in format_examples:
+        name = format_data.get('name')
+        reference = format_data.get('reference')
+        for example in format_data.get('examples', []):
+            skip = example.get('skip')
+            if not skip:
+                filename = example.get('filename')
+                url = example.get('url')
+                extension = filename.split('.')[-1]
+                filepath = Path(EXAMPLES_FOLDER, filename)
+                print(f'Evaluating {filename}. ')
+                for tilesource_name, readable in large_image.canReadList(filepath):
+                    tilesource = available_tilesources.get(tilesource_name)
+                    if readable and tilesource:
+                        try:
+                            s = tilesource(filepath)
+                            results.append(
+                                dict(
+                                    name=name,
+                                    reference=reference,
+                                    extension=extension,
```

Comment:
When there are multiple common extensions, should we list them here?  (e.g. tif and tiff, jp2 and j2k, etc)?

Reply:
Changed extension column to show a list of possible extensions for the type (instead of only getting the extension from the example file): [f11383b](https://github.com/girder/large_image/pull/1622/commits/f11383b91889ec1b1a95b1495d6b704e6b9f0d4a)